docs/usage/create-delta-lake-table.md

```diff
@@ -0,0 +1,26 @@
+# Creating a Delta Lake Table
+
+This section explains how to create a Delta Lake table.
+
+You can easily write a DataFrame to a Delta table.
+
+```python
+import deltalake as dl
```

Comment:
I think you forgot to remove this import as well

Reply:
Updated.